<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tables</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="PushButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RemoveButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SearchButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>60</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Найти</string>
    </property>
   </widget>
   <widget class="QComboBox" name="TableSelector">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Неупорядоченная таблица (массив)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Неупорядоченная таблица (список)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Упорядоченная таблица (список)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Хэш-таблица (список)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>АВЛ-дерево</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="StatusLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="ValueInputField">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Значение</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="KeyInputField">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Ключ</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>611</width>
      <height>441</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Индекс</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ключ</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Значение</string>
     </property>
    </column>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <attribute name="headerVisible">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
